uat/custom-components/README.md

```diff
@@ -0,0 +1,22 @@
+# Test MQTT5 client bases on AWS IoT Device SDK for Java v2
```

Comment:
We going to have several MQTT clients. 
Does it mean we will move SDK-based client one more time ?

Reply:
Project structure was updated according to discussed solution ba76450deb76f111510165f55bac5a57e3d60a8e